### ISSUE TITLE
v0.21: startup message states that telemetry is enabled in dev mode

### ISSUE THREAD

**Issue by guimachiavelli** [meilisearch/MeiliSearch#1558]
**Describe the bug**
MeiliSearch does not collect data in development mode, but this is the startup message a user gets when launching a local instance with `--env=development`:

```
[…]

Database path:		"./data.ms"
Server listening on:	"http://127.0.0.1:7700"
Environment:		"development"
Commit SHA:		"71e1cb472f89b2e277f4daf2857d67fd27072433"
Commit date:		"2021-07-01"
Package version:	"0.21.0"

Thank you for using MeiliSearch!

We collect anonymized analytics to improve our product and your experience. To learn more, including how to turn off analytics, visit our dedicated documentation page: https://docs.meilisearch.com/reference/features/configuration.html#analytics

Anonymous telemetry:   "Enabled"

[…]
```

The last line states telemetry is enabled.

**To Reproduce**
1. Launch an instance with `--env=development` or `MEILI_ENV=development`

**Expected behavior**
`Anonymous telemetry` should display as `"Disabled"`

**MeiliSearch version:** v0.21.0 RC3

**Comment by sanders41** [comment]
`--no-alalytics=true` will disable it

**Comment by guimachiavelli** [comment]
Thanks for that, @sanders41, but after a quick chat with @curquiza we realised that this is actually an issue with the upcoming v0.21 docs. 

On the second paragraph of https://dev.docs.meilisearch.com/learn/what_is_meilisearch/telemetry.html, we state that instances running with `--env=development` have telemetry disabled by default. It turns out this is wrong, so we'll fix it on that repo.

Closing the issue.
